tests/integration/istio/step1/index.ts

```diff
@@ -0,0 +1,45 @@
+import * as pulumi from "@pulumi/pulumi";
+import * as k8s from "@pulumi/kubernetes";
+
+import { k8sProvider, k8sConfig, k8sCluster } from "./cluster";
+import { kubeInject, istio } from "./istio";
+
+new k8s.core.v1.Namespace(
+    "bookinfo",
+    { metadata: { name: "bookinfo", labels: { "istio-injection": "enabled" } } },
+    { provider: k8sProvider }
+);
+
+function addNamespace(o: any) {
+    if (o !== undefined) {
+        o.metadata.namespace = "bookinfo";
+    }
+}
+
+const bookinfo = new k8s.yaml.ConfigGroup(
```

Comment:
Is `ConfigGroup` rather than `ConfigFile` needed because of istio?

Reply:
I forgot about this, this is from when we were reading the file manually and using `istioctl` to inject the sidecar.